cpp/modmesh/python/common.hpp

```diff
@@ -154,6 +154,9 @@ namespace modmesh
 namespace python
 {
 
+#pragma warning(push)
+#pragma warning(disable : 4251) // needs to have dll-interface to be used by clients of class
```

Comment:
just ignore it, so far we don't need to care about it

Reply:
clang-tidy doesn't like the pragma that it does not recognize: https://github.com/solvcon/modmesh/actions/runs/3354863488/jobs/5558705457#step:9:223 

What about fencing the MSC pragmas using `_MSC_VER` like in https://github.com/solvcon/modmesh/blob/d690bacf2d076b9b7423031776d8c71bbbfc133c/cpp/modmesh/base.hpp#L42